scripts/lib.star

```diff
@@ -906,11 +930,13 @@ def get_windows_steps(edition, ver_mode, is_downstream=False):
     else:
         source_commit = ' $$env:SOURCE_COMMIT'
 
+    init_cmds = []
     sfx = ''
     if edition == 'enterprise':
         sfx = '-enterprise'
-    init_cmds = []
-    if edition != 'enterprise':
+    elif edition == 'enterprise2':
+        sfx = '-enterprise2'
+    else:
```

Comment:
I don't think we need this step for 'enterprise2'.
But if we do, this can be simplified with something like: 
```
if edition in ('enterprise', 'enterprise2')
    sfx = '-' + edition
```

Reply:
@AgnesToulet Will check.

Reply:
@AgnesToulet Thanks, applied your simplification. I agree the step isn't used for enterprise2, but it''s best to be robust in case the assumption should become invalid.